Remotion/ObjectBinding/Web/UI/Controls/BocCompoundColumnDefinition.cs

```diff
@@ -80,7 +81,7 @@ protected override void OnOwnerControlChanged ()
     [PersistenceMode (PersistenceMode.Attribute)]
     [Category ("Format")]
     [Description ("A format string describing how the values accessed through the Property Path are merged.")]
-    [DefaultValue ("")]
+    [DefaultValue (typeof (WebString), "")]
```

Comment:
```suggestion
    [DefaultValue ("")]
```

fixed changeset typo